src/fdcache_entity.h

```diff
@@ -95,6 +105,7 @@ class FdEntity
         int GetPhysicalFd() const { return physical_fd; }
         bool IsModified() const;
         bool MergeOrgMeta(headers_t& updatemeta);
+        int UploadPending(int fd = -1);
```

Comment:
Better to remove the default parameter and just let the caller specify -1 when needed?

Reply:
Since `fd` was omitted only when calling from inside the `FdEntity` class, there is no problem by specifying `-1`.
Delete the default value.